integrations/docker/images/chip-build-mbedos/Dockerfile

```
@@ -0,0 +1,60 @@
+ARG VERSION=latest
+FROM connectedhomeip/chip-build:${VERSION}
+
+# ------------------------------------------------------------------------------
+# Install system tools via apt
+RUN set -x \
+    && apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -fy \
+    python3-setuptools \
+    python3-usb \
+    software-properties-common \
+    build-essential \
+    astyle \
+    mercurial \
+    && rm -rf /var/lib/apt/lists \
+    && : # last line
+
+# ------------------------------------------------------------------------------
+# Install Mbed-OS sources
+RUN set -x \
+    && (mkdir -p /opt/mbed-os \
+    && cd /opt/mbed-os \
+    && wget https://github.com/ARMmbed/mbed-os/archive/mbed-os-6.6.0.tar.gz \
```

```suggestion
    && wget --progress=dot:giga https://github.com/ARMmbed/mbed-os/archive/mbed-os-6.6.0.tar.gz \
```
`wget` tends to produce tons of useless dotted progress output. You can limit that with the `giga` param.

---

Done.